conda/environment.yml

```diff
@@ -12,4 +12,4 @@ dependencies:
   - pip
   - pip:
     - qualname>=0.1.0
-    - keras_preprocessing>=1.1.2
+    - keras_preprocessing>=1.1.2
```

Comment:
can you add a newline to every file at the end of which you can see the ⛔️ symbol?